ouroboros-consensus-cardano/changelog.d/20231110_165234_carl.hammann_vote_delegatee_query.md

```diff
@@ -0,0 +1,24 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+
+### Non-Breaking
+
+- Add a query for vote delegatees: GetFilteredVoteDelegatees
```

Comment:
Hmm, given that CI in upstream packages usually uses `-Werror`, I think it makes sense to actually mark this as "Breaking" (though we seem to have been a bit inconsistent in the [past](https://github.com/input-output-hk/ouroboros-consensus/blob/main/ouroboros-consensus-cardano/CHANGELOG.md)).